paste.c

```diff
@@ -8,7 +8,7 @@ int wmain() {
 	HANDLE clip = GetClipboardData(CF_UNICODETEXT);
 	if (clip) {
 		_setmode(_fileno(stdout), _O_U8TEXT);
-		wprintf(L"%s\n", clip);
+		wprintf(L"%s", clip);
```

Comment:
Is it correct, yes.

Also, wide chars != unicode. Don't be fooled. Windows has kind of terrible support for unicode.